pyat/at/lattice/lattice_object.py

```diff
@@ -50,6 +50,7 @@
         ('collective_pass', elt.Collective, 'auto'),
         ('diffusion_pass', elt.QuantumDiffusion, 'auto'),
         ('energyloss_pass', elt.EnergyLoss, 'auto'),
+        ('simplequantdiff_pass', elt.SimpleQuantDiff, 'auto')
```

Comment:
May diffusion_pass, energyloss_pass, simplequantdiff_pass can be grouped into a new radiative_pass argument?

Reply:
To be discussed. I am not sure how best to do it

Reply:
Ok for now